### ISSUE TITLE
ContentPacks are not compatible with GrayLog3

### ISSUE THREAD

**Issue by bager13** [dio99/graylog#1]
I tried to import JunOS and PulseSecure Content pack into GrayLog 3.0.2 but I get the following error:

> Error importing content pack, please ensure it is a valid JSON file. Check your Graylog logs for more information.<br /><small>Unable to map property extractors. Known properties include: entities, rev, summary, vendor, inputs, lookup_tables, name, requires, url, outputs, streams, lookup_data_adapters, lookup_caches, _id, grok_patterns, description, category, dashboards, parameters, v, id</small>

I checked some JSON from some content packs and it seems there are quite some differences so I guess it has not been updated yet for GrayLog3.

**Comment by dio99** [comment]
yes true this was created in old version of graylog it does not work to import in version 3.x
im gonna see if i get some time to redo the them and export them in version 3.1
// anders
